hoodie-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -0,0 +1,61 @@
+{
+   "namespace":"com.uber.hoodie.avro.model",
+   "type":"record",
+   "name":"HoodieCommitMetadata",
+   "fields":[
+      {
+         "name":"partitionToWriteStats",
+         "type":["null", {
+            "type":"map",
+            "values":{
+               "type":"array",
+               "items":{
+                  "name":"HoodieWriteStat",
+                  "type":"record",
+                  "fields":[
+                     {
+                        "name":"fileId",
+                        "type":"string"
+                     },
+                     {
+                        "name":"fullPath",
+                        "type":"string"
+                     },
+                     {
+                        "name":"prevCommit",
+                        "type":"string"
+                     },
+                     {
+                        "name":"numWrites",
+                        "type":"string"
+                     },
+                     {
+                        "name":"numDeletes",
+                        "type":"string"
+                     },
+                     {
+                        "name":"numUpdateWrites",
+                        "type":"string"
+                     },
+                     {
+                        "name":"totalWriteBytes",
+                        "type":"string"
+                     },
+                     {
+                        "name":"totalWriteErrors",
+                        "type":"string"
+                     }
+                  ]
+               }
+            }
+         }]
+      },
```

Comment:
FilePaths is essentially derived from the partitionToWriteStats in HoodieCommitMetadata.